certbot/cli.py

```diff
@@ -1133,6 +1133,10 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         ["testing", "standalone"], "--http-01-address",
         dest="http01_address",
         default=flag_default("http01_address"), help=config_help("http01_address"))
+    helpful.add(
```

Comment:
Why are these changes in both this PR and #6857? This value or `--tls-sni-01-port` doesn't seem to be used in the Apache plugin so I think these changes should be removed from this PR and left in the Nginx one.

Reply:
Yes I did not remove it yet, after seeing that it was not useful for Apache.